src/ontology/uberon.Makefile

```diff
@@ -209,163 +219,213 @@ mirror-ehdaa2: | $(TMPDIR)
-# The corresponding rules are invoked during the release process (as
-# prerequisites of the composite-* pipeline), but normally both MIR and
-# IMP are false at this point, so no commands are executed. The rules
-# themselves are necessary to trigger the cloning of the
-# developmental-stage-ontologies repository, though.
-# Step 1: Map EMAPA stage IDs to MmusDv
-$(TMPDIR)/fixed-emapa.obo: $(TMPDIR)/mirror-emapa.obo
-	if [ $(MIR) = true ] && [ $(IMP) = true ]; then \
-		$(SCRIPTSDIR)/obo-grep.pl -r 'id: EMAPA' $< | \
-		$(SCRIPTSDIR)/fix-emapa-stages.pl > $@; \
-	fi
-
-# Step 2: Merge with MmusDv to create the final, fixed mirror
-mirror/emapa.owl: $(TMPDIR)/fixed-emapa.obo $(TMPDIR)/update-stages
-	if [ $(MIR) = true ] && [ $(IMP) = true ]; then \
-		$(ROBOT) merge -i $(TMPDIR)/fixed-emapa.obo \
-		               -i $(TMPDIR)/developmental-stage-ontologies/src/mmusdv/mmusdv.obo \
-		         convert -f ofn -o $@ ; \
-	fi
+## ONTOLOGY: ssso
+.PHONY: mirror-ssso
+.PRECIOUS: $(MIRRORDIR)/ssso.owl
+mirror-ssso: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(IMP) = true ]; then curl -L https://github.com/obophenotype/developmental-stage-ontologies/releases/latest/download/ssso-merged-uberon.obo --create-dirs -o $(TMPDIR)/mirror-ssso.obo --retry 4 --max-time 200 && \
+		$(ROBOT) convert -i $(TMPDIR)/mirror-ssso.obo -o $(TMPDIR)/$@.owl; fi
+
+## ONTOLOGY: mmusdv
+.PHONY: mirror-mmusdv
+.PRECIOUS: $(MIRRORDIR)/mmusdv.owl
+mirror-mmusdv: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(IMP) = true ]; then curl -L https://github.com/obophenotype/developmental-stage-ontologies/releases/latest/download/mmusdv.owl --create-dirs -o $(MIRRORDIR)/mmusdv-download.owl --retry 4 --max-time 200 && \
+		$(ROBOT) convert -i $(MIRRORDIR)/mmusdv-download.owl -o $(TMPDIR)/$@.owl; fi
+
+## ONTOLOGY: hsapdv
+.PHONY: mirror-hsapdv
+.PRECIOUS: $(MIRRORDIR)/hsapdv.owl
+mirror-hsapdv: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(IMP) = true ]; then curl -L https://github.com/obophenotype/developmental-stage-ontologies/releases/latest/download/hsapdv.owl --create-dirs -o $(MIRRORDIR)/hsapdv-download.owl --retry 4 --max-time 200 && \
+		$(ROBOT) convert -i $(MIRRORDIR)/hsapdv-download.owl -o $(TMPDIR)/$@.owl; fi
 
 
 # ----------------------------------------
 # "LOCAL" IMPORTS
 # ----------------------------------------
+ifeq ($(strip $(IMP)),true)
 
 # The following imports are used in Uberon-specific pipelines. They are
 # *not* updated as part of the refresh-imports pipeline. In fact, they
 # are normally never updated automatically, since Uberon-specific
 # pipelines are typically run with IMP set to false.
 
-# Map legacy OBO-format object properties to their BFO/RO equivalent
-# FIXME: this may no longer be needed for some of these imports
-# (https://github.com/obophenotype/uberon/issues/3017)
+# By default, we would like to simply use the mirrored ontology as it
+# is, but to avoid injecting random crap, we make a pseudo-base.
```

Comment:
```suggestion
# is, but to avoid injecting axioms that do not belong to the ontology and are potentially conflicting, we make a pseudo-base.
```

😆

Reply:
I don’t see any difference, the two sentences seem completely equivalent to me. :D